### ISSUE TITLE
Bug: using mathjax with defaults adds wrong script url [2.10.1]

### ISSUE THREAD

**Issue by gabyx** [jgm/pandoc#6593]
Reproduce:

*test.yaml*:
```yaml
html-math-method:
    method: mathjax
```

```shell
echo '$asdasd$' | pandoc -f markdown+tex_math_dollars --standalone --defaults=test.yaml -t html5
```

adds a wrong url
```html
<script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/" type="text/javascript"></script>
```

**However**:
```shell
echo '$asdasd$' | pandoc -f markdown+tex_math_dollars --standalone --mathjax -t html5
```
adds the correct url `https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js`

**Comment by mb21** [comment]
I believe this is fixed in pandoc 2.10

**Comment by gabyx** [comment]
Thx.
I will test it.

**Comment by jgm** [comment]
I think the issue is still here, if it is an issue.
(Note: triggering mathjax via yaml metadata isn't an officially supported feature; you're relying on some fragile assumptions about how the writers work.)

**Comment by jgm** [comment]
Oh, never mind, I see you're using a defaults file. That *should* work, so this is a bug.

**Comment by gabyx** [comment]
Still an issue in 2.10.1, yes, tested it.

**Comment by gabyx** [comment]
Probably relates to this [line](https://github.com/jgm/pandoc/blob/a5fa55969f1b4afc0ca3e38be50b69c65d43a460/src/Text/Pandoc/Options.hs#L111)?

**Comment by jgm** [comment]
Closed by d8ad766d17603784b86fc5c2e1b22864125d04d1
